metadata.json

```diff
@@ -7,7 +7,10 @@
   "source": "git@github.com/puppetlabs/puppetlabs-vault",
   "project_page": "https://github.com/puppetlabs/puppetlabs-vault",
   "dependencies": [
-
+    {
+      "name":"puppetlabs-ruby_task_helper",
+      "version_requirement":">= 0.4.0 < 1.0.0"
```

Comment:
I think we're on 0.5.1 https://forge.puppet.com/puppetlabs/ruby_task_helper is there a reason it needs to be 0.4.0?

Reply:
@lucywyman This module doesn't depend on 0.5.1 and is compatible with 0.4.0, so I think leaving the version range less restrictive makes sense.

That said, puppetlabs/puppetlabs-azure_inventory uses debug statements, which are only available in 0.5.1, so I'll update that one.